.pre-commit-config.yaml

```diff
@@ -10,26 +10,26 @@ repos:
 
 # Autoremoves unused imports
 - repo: https://github.com/hadialqattan/pycln
-  rev: v1.1.0
+  rev: v2.4.0
```

Comment:
Just out of curiosity, but I am wondering if you are going to use automatic updates with [pre-commit.ci](https://pre-commit.ci/)?

Reply:
Thank you @tkoyama010 for taking the time to look at this!  
I actually am avoiding pre-commit right now because it slows me down to learn yet another tool.  
But I will keep in mind that it can do automatic updates, I really appreciate the reminder.